src/Core/Models/Api/Response/SendResponseModel.cs

```diff
@@ -66,5 +67,6 @@ public SendResponseModel(Send send, GlobalSettings globalSettings)
         public DateTime RevisionDate { get; set; }
         public DateTime? ExpirationDate { get; set; }
         public DateTime DeletionDate { get; set; }
+        public bool? HideEmail { get; set; }
```

Comment:
This can be `bool` here in the View Model because you're defaulting the value. It's only the data model that needs to match the table definition (`bool?` = `BIT NULL`)

```suggestion
        public bool HideEmail { get; set; }
```